app/decorators/guide_decorator.rb

```diff
@@ -1,6 +1,19 @@
-class GuideDecorator < Draper::Decorator
-  delegate :id, :slug, :concise_label, :description,
-           :option?, :related_to_appointments?, :related_to_booking?
+class GuideDecorator
+  delegate(
+    :id,
+    :slug,
+    :concise_label,
+    :description,
+    :option?,
+    :related_to_appointments?,
+    :related_to_booking?,
+    :==,
+    to: :guide
+  )
```

Comment:
It was considered, but I decided against. Adding `SimpleDelegator` in the inheritance chain, with this being the base decorator for the govspeak and HTML decorators would mean this family of decorators forward absolutely everything. Also bearing in mind I've only added one more target: `:==` and generally I favour the explicitness of knowing right up front exactly which calls are forwarded - as is afforded by the `delegate` call.